torch/csrc/jit/passes/quantization.cpp

```diff
@@ -446,7 +446,8 @@ class InsertObserversHelper {
       bool is_entry_point = false,
       std::unordered_set<Value*> graph_observed_values =
           std::unordered_set<Value*>());
-
+  // Is dynamic quantization enabled for the observer pass.
+  bool is_dynamic = false;
```

Comment:
maybe make this private and expose this through a function like `set_dynamic_flag`